### DISCUSSION TITLE
Pages Not Sorting By API Tags

### DISCUSSION THREAD

**Discussion by BrydonP** [PaloAltoNetworks/docusaurus-openapi-docs#391] [Q&A]
Hello, As stated above, my pages refuse to sort in the sidebar by their API tags. 

I have confirmed I have tags in my json file and I have added the config to group the paths by tag in the docusaurus.config.js file as well. 

`sidebarOptions: {
              groupPathsBy: "tag",
              categoryLinkSource: "tag"
            },`

I know there is some configuration needs to be done in the sidebar.js, but it was my understanding that it's done using the auto-generated model based on directory. Or is the addon using some kind of tagging method for the sorting? Unclear on the methodology being used here. 

I would appreciate any help as this seems to be the last step in getting this fully working!

**Comment by sserrata** [comment]
Hi @BrydonP, the plugin should have auto-generated a `sidebar.js` slice/file (in the same folder as the generated MDX). You'll want to reference that `sidebar.js` slice in your full `sidebars` definition file, like in the demo: https://github.com/PaloAltoNetworks/docusaurus-openapi-docs/blob/main/demo/sidebars.js#L61
